src/open_inwoner/cms/cases/views/status.py

```diff
@@ -172,6 +172,15 @@ def get_context_data(self, **kwargs):
             # The end status data is not passed if the end status has been reached,
             # because in that case the end status data is already included in `statuses`
             end_statustype = next((s for s in statustypen if s.is_eindstatus), None)
```

Comment:
Wouldn't this line become irrelevant after the changes you made? If I understand it correctly, the conditional below would always be triggered and override `end_statustype` to be the statustype of the last status (if `is_eindstatus` is `True`) right?

Reply:
I indeed went for the most minimal change for my hotfix for v1.12. I suspect you're correct and have updated this branch to remove this line and avoid future confusion

Reply:
Hmm okay I may have been wrong, I glanced over that fact that in this line the `statustypen` are used directly, so that way we will still get the `end_statustype` even if this status hasn't been reached yet, so I think we still need this line :sweat_smile: